### ISSUE TITLE
Active links from the changeset info window

### ISSUE THREAD

**Issue by jfriedel** [OpenGrok/OpenGrok#1367]
The links otherwise available from the history page of a file in the repo (e.g. links to architecture review cases, or links to bug databases) should be available also from the the window showing up in the changeset info window (annotation: ON).

![active_links](https://cloud.githubusercontent.com/assets/5879789/22465280/34249d6a-e7bb-11e6-9c62-1ec457cadbaa.jpg)

**Comment by tulinkry** [comment]
This also requires that the tooltip must be accessible with mouse

**Comment by vladak** [comment]
Like with #1366 , the idea of hunting for the tooltip with mouse is not very intriguing to me, to say the least.

**Comment by jfriedel** [comment]
"hunting for the tooltip"?..  Anyway, how about a alternate solution -- is there a way of how to get quickly to the history page of the given file and land on the that page on the given changeset? That said, instead of remembering the changeset id, then going to the history page and lookup that changeset id on that page, one would be able to quickly get to the given line on the history page. On the history page, the links already work.

**Comment by vladak** [comment]
you mean the link name ? that already works, e.g. https://grok/source/history/opengrok-master/src/org/opensolaris/opengrok/index/Indexer.java#b528c008

**Comment by jfriedel** [comment]
Yes, but how to get quickly from a source page with annotation on to the respective line on the history page?  I.e. how to quickly get

from here: https://grok/source/xref/opengrok-master/src/org/opensolaris/opengrok/index/Indexer.java?a=true
to here: https://grok.cz.oracle.com/source/history/opengrok-master/src/org/opensolaris/opengrok/index/Indexer.java?n=25&start=50#5e6c91d7 (note: see that this is not the first history page..)
